doc/reference_processing.xml

```diff
@@ -2403,6 +2403,58 @@ FROM (SELECT  'POLYGON((0 0, 8 8, 0 16, 0 0))'::geometry geom) As foo;
           </refsection>
     </refentry>
 
+
+    <refentry id="ST_SimplifyPolygonal">
+      <refnamediv>
+        <refname>ST_SimplifyPolygonal</refname>
+        <refpurpose>Computes a boundary-respecting hull of a polygonal geometry, with hull shape determined by a target parameter.</refpurpose>
+      </refnamediv>
+
+      <refsynopsisdiv>
+        <funcsynopsis>
+         <funcprototype>
+            <funcdef>geometry <function>ST_SimplifyPolygonal</function></funcdef>
+            <paramdef><type>geometry </type> <parameter>param_geom</parameter></paramdef>
+            <paramdef><type>float </type> <parameter>vertex_fraction</parameter></paramdef>
```

Comment:
is that a useful call signature? I can imagine having vertex_count if I want to try this instead of ST_Subdivide in some place. I can see going down to some threshold in either area or vertex count, but not really a case for "let's remove 10% points from each geometry, and yes just one point from this 10-point one".

Reply:
I think the question of "how much" is hard to nail down, different people will have different takes. "Somewhat less detailed" for some and "no more than this many vertices" for others.

Reply:
In Postgis we're normally working on sets of map objects, unlike GEOS that works on one. A thing that feels natural to me to expect from "simplification" is that things that are simple already aren't touched, and I believe this assumption is not followed with this signature. 

Do I get it right that the actual controllable range is between ST_NPoints(geom) and ST_NPoints(ST_ConvexHull(geom)) for outer hull and ST_NPoints(geom) and 3 for inner hull?